liblangutil/Token.h

```diff
@@ -313,7 +313,7 @@ namespace TokenTraits
 
 	constexpr bool isEtherSubdenomination(Token op) { return op == Token::SubWei || op == Token::SubSzabo || op == Token::SubFinney || op == Token::SubEther; }
 	constexpr bool isTimeSubdenomination(Token op) { return op == Token::SubSecond || op == Token::SubMinute || op == Token::SubHour || op == Token::SubDay || op == Token::SubWeek || op == Token::SubYear; }
-	constexpr bool isReservedKeyword(Token op) { return (Token::Abstract <= op && op <= Token::Virtual); }
+	constexpr bool isReservedKeyword(Token op) { return (Token::Abstract <= op && op <= Token::Unchecked); }
```

Comment:
If you want you can move ``Abstract`` out of the reserved keywords as well here - otherwise we should do it separately - apparently that was missed in the abstract PR. Or is this one just not rebased on top of that one?

Reply:
I believe it was forgotten. I'll move it.